src/testdrive/src/parser.rs

```diff
@@ -115,13 +115,13 @@ fn parse_builtin(line_reader: &mut LineReader) -> Result<BuiltinCommand, InputEr
             }
         };
         lazy_static! {
-            static ref VALID_KEY_REGEX: Regex = Regex::new("^[a-z0-9\\-]*$").unwrap();
+            static ref VALID_KEY_REGEX: Regex = Regex::new("^[a-z0-9_\\-\\.\\*]*$").unwrap();
         }
         if !VALID_KEY_REGEX.is_match(pieces[0]) {
             return Err(InputError {
                 msg: format!(
                     "invalid builtin argument name '{}': \
-                     only lowercase letters, numbers, and hyphens allowed",
+                     only lowercase letters, numbers, hyphens, underscores, and stars allowed",
```

Comment:
This bit seems funny. This is about what's allowed in the "key" part of the command if I remember right, so the `command=` part, not the part that follows. I don't think that ought to allow underscores and stars. And indeed your test file parses correclty without this bit of the change!

Reply:
you are right! I have removed this change.